dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@ protected void postProcessAfterScopeModelChanged(ScopeModel oldScopeModel, Scope
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Issue**: Adding null check 'registryConfig != null' before accessing 'registryConfig.getScopeModel()' enhances the robustness of the code. However, it's essential to ensure that this additional check aligns with the expected logic and does not hide potential issues where 'registryConfig' should not be null. <br> **Fix**: Validate that the added null check is indeed required and does not mask a scenario where a null 'registryConfig' indicates a misconfiguration or a bug elsewhere in the code. <br> **Code Suggestion**: 
 ```
 Validate that the added null check is indeed required and does not mask a scenario where a null 'registryConfig' indicates a misconfiguration or a bug elsewhere in the code.
 ```